README.md

```diff
@@ -74,7 +74,7 @@ Step into newly created `djangogirls` directory:
 cd djangogirls
 ```
 
-Create a new virtual environment (python 3.9) if needed. Then, install all the required dependencies.
+Create a new virtual environment (Python 3.10) if needed. Then, install all the required dependencies.
```

Comment:
For now our image supports up to `Python 3.9` but I could just upgrade the image to the later one that supports `Python 3.10`.

Reply:
@amakarudze At this point, it might be worth waiting for that. Once we've got the active PRs/issues completed, I'm curious if the project can be moved to django 4.1 and python 3.11 - this is probably largely down to if the dependencies of the project support these.

Using `pip-tools` is the quickest way to find if any dependency will prevent the use of django 4.1. Then `python-upgrade` and `django-upgrade` can be used to update the codebase automatically - and they can be added to the `pre-commit` config to ensure code continues to meet the target python & django versions.

I may have some time to look at this in the next few weeks - but I start a new job on Feb 13th so I'll get very busy around then as I try to get used to all that comes from a new job!

(fyi @vanessa @ramonsaraiva)

Reply:
Noted @marksweb. Let me work on updating the image and then let everyone know once that is done. 

@vanessa I am going to wait on both of your PRs till I upgrade deployment to Python 3.10 then look at them. Thanks for all your work @vanessa, @marksweb, @ramonsaraiva. Very much appreciated.

Reply:
Makes total sense!

Reply:
@marksweb @vanessa @ramonsaraiva We are now using Python 3.10 in production so I will look at this PR today before I merge so I know what changes I am merging in to the repo .